holidays/countries/turkmenistan.py

```diff
@@ -0,0 +1,69 @@
+from holidays.calendars.gregorian import JAN, MAR, MAY, SEP, OCT, DEC
+from holidays.groups import InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Turkmenistan(HolidayBase, InternationalHolidays, IslamicHolidays):
+    """
+    Turkmenistan holidays.
+    References:
+        https://en.wikipedia.org/wiki/Public_holidays_in_Turkmenistan
+        https://www.timeanddate.com/holidays/turkmenistan/
+    """
+
+    country = "TM"
+    start_year = 1991 
+
+    def __init__(self, *args, **kwargs):
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self) 
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        """Add public holidays for Turkmenistan."""
+        if self._year >= 1991:
```

Comment:
You can safely remove the docstring here.

Also, `start_year = [YEAR]` above already acts as `if self._year >= [YEAR]` for all holidays assignment.